COPY/usr/lib/systemd/system/evmserverd.service

```diff
@@ -6,13 +6,11 @@ Wants=memcached.service
 [Service]
 WorkingDirectory=/var/www/miq/vmdb
 EnvironmentFile=/etc/default/manageiq*.properties
-ExecStart=/bin/sh -c "/bin/evmserver.sh start"
-ExecStop=/bin/sh -c "/bin/evmserver.sh stop"
+Environment=EVMSERVER=true
+ExecStart=/usr/bin/ruby lib/workers/bin/evm_server.rb
```

Comment:
This only runs start... will this cause any loss of behavior, specifically, needing to run `status` or the update start|stop directly?

Reply:
`systemctl status` didn't use `/bin/evmserver.sh status` and after this dep PR (https://github.com/ManageIQ/manageiq-rpm_build/pull/185) we won't need update_{start,stop} anymore.

Reply:
yeah, I think the `status` was a relic of the old init style.  Thanks!